RadixWallet/Features/TransactionReviewFeature/TransactionReview.swift

```diff
@@ -1235,13 +916,13 @@ enum FeeValidationOutcome {
 extension ReviewedTransaction {
 	var feePayingValidation: Loadable<FeeValidationOutcome> {
 		feePayer.map { selected in
-			switch transaction {
-			case .nonConforming, .conforming(.accountDepositSettings):
-				return selected.validateBalance(forFee: transactionFee)
 
-			case let .conforming(.general(generalTransaction)):
+			switch executionSummary.detailedManifestClass {
+			case .none: // Non-conforming
+				return selected.validateBalance(forFee: transactionFee)
+			case .general:
```

Comment:
we should also handle `.transfer` and `.poolContribution`(users can withdraw XRD to contribute to pool)

Reply:
got it